ee/frontend/mobile-replay/transformers.ts

```diff
@@ -121,6 +125,106 @@ function makeImageElement(wireframe: wireframeImage, children: serializedNodeWit
     }
 }
 
+function inputAttributes<T extends wireframeInputComponent>(wireframe: T): attributes {
+    const attributes = {
+        style: makeStylesString(wireframe),
+        type: wireframe.inputType,
+        ...(wireframe.disabled ? { disabled: wireframe.disabled } : {}),
+    }
+
+    switch (wireframe.inputType) {
+        case 'checkbox':
+            return {
+                ...attributes,
+                ...(wireframe.checked ? { checked: wireframe.checked } : {}),
+            }
+        case 'radio':
+            return {
+                ...attributes,
+                ...(wireframe.checked ? { checked: wireframe.checked } : {}),
+                // radio value defaults to the string "on" if not specified
+                // we're not really submitting the form, so it doesn't matter 🤞
+                // radio name is used to correctly uncheck values when one is checked
+                // mobile doesn't really have it, and we will be checking based on snapshots,
+                // so we can ignore it for now
+            }
+        case 'button':
+            return {
+                ...attributes,
+                ...(wireframe.value ? { value: wireframe.value } : {}),
+            }
+        case 'text_area':
+            return {
+                ...attributes,
+                ...(wireframe.value ? { value: wireframe.value } : {}),
+            }
+        default:
+            return {
+                ...attributes,
+                ...(wireframe.value ? { value: wireframe.value } : {}),
+            }
```

Comment:
What would be rendered if no `inputType` is set? I wonder if it'd be better to not return anything.

Reply:
nope, it spat out nonsense 🙈 

great catch!